dlrs/clearlinux/tensorflow/oss/README.md

```diff
@@ -1,7 +1,6 @@
-## Deep Learning Reference Stack with Tensorflow and Optimized Eigen
+## Deep Learning Reference Stack with Pytorch and Intel® MKL-DNN
```

Comment:
Pytorch and Intel MKL ? shouldn't this be TensorFlow and optimized Eigen ?, also please change all Intel MKL-DNN to oneDNN  as seen here: https://github.com/oneapi-src/oneDNN